_data/authors.yml

```diff
@@ -27,3 +27,12 @@ newstudent:
   twitter: 
   google:
     plus: 
+
+pilarvillanueva:
+  name: Pilar Villanueva
+  email: mdv769@utexas.edu
+  bio: Graduate Student, Department of Spanish and Portuguese
+  avatar: image.jpeg
```

Comment:
Did you upload a file (like a photograph of yourself) called "image.jpeg"? If not, this should be left empty!

Reply:
I didn't, I am sorry. I thought I need to write that down.
I will try to fix it now.
Thank you!!